win_etw_provider/src/provider.rs

```diff
@@ -177,13 +177,8 @@ impl Provider for EtwProvider {
 
 #[inline(never)]
 fn write_failed(error: u32) {
-    #[cfg(feature = "std")]
-    {
-        println!("EventWrite failed: {}", error);
-    }
-    #[cfg(not(feature = "std"))]
-    {
-        let _ = error;
+    if cfg!(feature = "dev") {
```

Comment:
These should probably be changed to:

```
#[cfg(feature = "dev")] {
  ...
}
```

instead of using `cfg!`. Reason being, the code as written right now will fail to compile in `#[no_std]` environments, even if the `cfg!` evaluates to `false`.

Reply:
I made this change, but then started getting warnings about some variables being unused if they were only used in the conditional block and `feature = "dev"` was unset. To work around this I renamed them to have a leading underscore, but if you have a different preference for how to handle this, let me know.